packages/ui/components/logo/Logo.tsx

```diff
@@ -1,20 +1,41 @@
+import { useSubdomainContext } from "@calcom/features/orgs/SubdomainProvider";
 import classNames from "@calcom/lib/classNames";
 import { LOGO_ICON, LOGO } from "@calcom/lib/constants";
 
-export default function Logo({ small, icon }: { small?: boolean; icon?: boolean }) {
+export default function Logo({
+  small,
+  icon,
+  inline = true,
+  className,
+}: {
+  small?: boolean;
+  icon?: boolean;
+  inline?: boolean;
+  className?: string;
+}) {
+  const { isSubdomain, logoSrc, subdomain } = useSubdomainContext();
   return (
-    <h3 className="logo inline ">
+    <h3 className={classNames("logo", inline && "inline", className)}>
       <strong>
-        {icon ? (
-          <img className="mx-auto w-9 dark:invert" alt="Cal" title="Cal" src={LOGO_ICON} />
-        ) : (
+        {isSubdomain && (
           <img
             className={classNames(small ? "h-4 w-auto" : "h-5 w-auto", "dark:invert")}
-            alt="Cal"
-            title="Cal"
-            src={LOGO}
+            alt={subdomain || "Cal"}
+            title={subdomain || "Cal"}
```

Comment:
We can use the subdomain as the alt text here to improve a11y.

"ikea.cal.com" will have the alt "ikea"